Fix take_screenshot in system specs

andreslucena opened the pull request:
#### :tophat: What? Why?

While working in system specs, I found a couple of things: 

1. The `take_screenshot` method wasn't returning the correct path. It was generating the images correctly but in the terminal wasn't showing the correct path, so you needed to find the image/HTML manually. 
2. Most of the code that we were using here doesn't make sense anymore in Rails 6.1. We were copying it from [a gem](https://gitlab.com/deivid-rodriguez/system_test_html_screenshots/) and as its [description](https://gitlab.com/deivid-rodriguez/system_test_html_screenshots/-/blob/master/system_test_html_screenshots.gemspec#L12) says: 

>  spec.description   = %q{Rails is not interested on adding HTML screenshots to their system tests, so this gem monkeypatches Rails to achieve that.}

That's not correct anymore, with Rails 6.1 this feature [was added](https://github.com/rails/rails/commit/c699a877ca39951ba29cb8f9785f4a3909784f44) with the environment variable `RAILS_SYSTEM_TESTING_SCREENSHOT_HTML`, allowing us to simplify a lot of this file. The only thing that we keep is the sanitization introduced on #8802

#### :pushpin: Related Issues

- Related to #8411 

#### Testing

1. Go to a system spec
2. Add a `take_screenshot` invocation
3. See that the image path is correct 

:hearts: Thank you!

ahukkanen commented:
Should we still enable this by default?

The current behavior is that if the test fails, it will save both the HTML and the screenshot.

Also another difference I noticed is that it does not print `file:///` in front of the filename which removes the feature that we cannot click the link directly from the console window to open it. I can personally live with this aspect but I was just thinking if we want to preserve current functionality?

ahukkanen commented:
For example, here we could set this setting enabled by default, similarly to:
https://github.com/decidim/decidim/blob/021f1674cc49619b15b40fea4667cce2c6bc3529/decidim-dev/lib/decidim/dev/test/base_spec_helper.rb#L5

andreslucena commented:
> The current behavior is that if the test fails, it will save both the HTML and the screenshot.

Ok, I agree, I added it. 

> Also another difference I noticed is that it does not print file:// in front of the filename which removes the feature that we cannot click the link directly from the console window to open it. I can personally live with this aspect but I was just thinking if we want to preserve current functionality?

Oh, I didn't see that, as my IDE (VS Code) already handles these types of links: it opens when clicked even if it doesn't have the file URI scheme. In my terminal emulator (terminator) that doesn't work, so I think that'd be nice to have it. 

I've added it in the output.

ahukkanen reviewed (approved):
:+1:
